### ISSUE TITLE
Make toolbar & tabbar icons svg to have them in higher definition but also have an option to have them as the cureent ones

### ISSUE THREAD

**Issue by ccuser44** [notepad-plus-plus/notepad-plus-plus#9380]
### Description of the Issue

The tabbar& toolbar icons are a bit pixelated.

### Expected Behavior

Theyvwould be SVG images.

### Actual Behavior

They were not SVG images and were pixelated instead.

I think you should add an option yo habe them as SVG images. But also include an option to have them as the current ones.

**Comment by mere-human** [comment]
In theory, this is nice to have.
However, in practice, there are some questions.

I suppose this will require tracing or upscaling the existing bitmap icons to make them into vector ones.

Also, I'm not sure the application supports rendering SVG icons. 
Rather than that, other apps usually have several versions of the same icon in different resolutions.

**Comment by AraHaan** [comment]
@mere-human not true, to make them into svg would mean to manually remake them using vector lines and painting and shit.

However with it being svg then one could render it at any size, convert them at runtime to have an DIB section, then recolor them at will on the whole thing if needed.

It's actually easy to have an SVG lib, and have functions where it converts it's data to an HBITMAP, then from there notepad++ could use gdi+ or whatever it does now to render it all.

**Comment by tomasz1986** [comment]
> However with it being svg then one could render it at any size

Just a comment, but being able to set custom sizes would be great in terms of accessibility. I have somewhat poor eyesight, so the small icons are a little bit difficult to use. The large ones, however, are way too big, basically gigantic, taking a lot of space on the screen as a result. I'd like to specify a size somewhat in the middle of the current ones.

**Comment by vertigo220** [comment]
I'm in total agreement with tomasz. Depending on screen size, distance, and resolution, the small icons range from big enough to borderline or even too small, but the big icons just make it look like they're for use for people with serious visual impairment and make it look bad and take up unnecessary space. The ideal size, IMO, would be somewhere in the middle, probably ~125-135% of the small icon size. It would be great to have another option, or at least have the icons in a folder in Program Files (I'm assuming they're currently embedded into the exe) so users can just manually resize them on their own.
